### DISCUSSION TITLE
Static Homepage with dynamic content (latest posts) -  theme "Technews (v.3.0.0.0)"?

### DISCUSSION THREAD

**Discussion by PechGehabt** [GetPublii/Publii#1775] [Help]
Hi!

I am using the latest version of Publii and the latest version of the theme "Technews (v.3.0.0.0)". I would like to use the "Homepage" feature to have more control over what is displayed on the homepage. But I would also like to have a dynamic part on that Homepage showing the latest X Posts. Any tip on how I can achieve that?

Thanks for you help!

**Comment by michaelfluegge** [comment]
I solved it this way: 

```
                        <h3Recent Posts</h3>
                        <ul>
                            {{#each @website.contentStructure.posts}}
                                {{#checkIf @index '<' 5}}
                                    <li><a href="{{url}}">{{title}}</a>
                                    </li>
                                {{/checkIf}}
                            {{/each}}
                        </ul>

**Reply by PechGehabt** [reply]
Hi! Thanks! How to put that into the homepage? If I add it as HTML in the editor it is interpreted as text. Do I have to change the template itself? I assume in index.hbs?

**Comment by michaelfluegge** [comment]
It might be a bit tricky for a beginner, and I am not a pro, yet. You have to create an override theme: https://getpublii.com/dev/theme-overrides/. And it is the index.hbs, you have to tweak, I think, but I am not sure. Maybe @candidexmedia can help?
